Application/EdFi.Ods.Api/Helpers/AssemblyLoaderHelper.cs

```diff
@@ -46,53 +46,53 @@ public static void LoadAssembliesFromExecutingFolder(bool includeFramework = fal
             // Storage to ensure not loading the same assembly twice and optimize calls to GetAssemblies()
             IDictionary<string, bool> loadedByAssemblyName = new ConcurrentDictionary<string, bool>();
 
-            LoadAssembliesFromExecutingFolder();
+            // Load referenced assemblies into the domain. This is effectively the same as EnsureLoaded in common

```

Comment:
This section of code was refactored out of a local function with the same name to improve readability